permissions/helpers.py

```diff
@@ -95,17 +81,15 @@ def can_modify_reservation_unit(user: AnyUser, reservation_unit: ReservationUnit
     return can_manage_units_reservation_units(user, reservation_unit.unit)
 
 
-def can_manage_service_sectors_applications(user: AnyUser, service_sector: ServiceSector) -> bool:
+def can_manage_service_sectors_applications(user: AnyUser) -> bool:
```

Comment:
This can be removed if the calls to `can_manage_service_sectors_applications` are changed in `ReservationUnitOptionPermission` and `AllocatedTimeSlotPermission`, so that update is permitted on them by virtue of the user having `CAN_HANDLE_APPLICATIONS` on the unit (of the reservation unit) of the option being updated or allocated.

Reply:
Done.